### ISSUE TITLE
add connection distribution

### ISSUE THREAD

**Issue by glommer** [scylladb/scylla-monitoring#241]
Looking at connections distribution per shard is becoming a daily routine for me when looking for problems.

We should add this to the dashboards. It belongs close to the other network statistics.

**Comment by tzach** [comment]
```
sum(scylla_transport_current_connections{instance=~"[[node]]", shard=~"[[shard]]"} ) by ([[by]]) 
```

**Comment by amnonh** [comment]
@glommer I think it can be closed by 17dee16f7ecc4b4c00b1608eeaa0e1512b6a3431
